.storybook/preview.tsx

```diff
@@ -16,93 +17,46 @@ import type { UtagContainer } from '@cmsgov/design-system';
 // Rewire analytics events to log to the console
 (window as UtagContainer).utag = { link: console.log };
 
-const customViewports = {
+const breakpointViewportSizes = {
   extraSmall: {
-    name: 'Extra Small - 320px',
+    name: '$media-width-xs',
     styles: {
       width: '320px',
       height: '800px',
     },
+    type: 'mobile',
```

Comment:
[Storybook docs say `type` is needed](https://storybook.js.org/docs/react/essentials/viewport#configuration) for the viewport objects, but they lie. Kept this in because it's not hurting anything.